source/handlers/PsychVideo.hx

```diff
@@ -0,0 +1,149 @@
+package handlers;
+
+import flixel.FlxSprite;
+import flixel.FlxCamera;
+import flixel.FlxG;
+import openfl.Assets as OpenFlAssets;
+#if sys
+import sys.FileSystem;
+#end
+#if VIDEOS_ALLOWED
+import VideoHandler;
```

Comment:
put hxcodec behind `VideoHandler`

Reply:
![image](https://user-images.githubusercontent.com/93128334/213768354-a7fc8b5d-3692-40ae-9152-833aba7ba76c.png)
can't be done

Reply:
it's `hxcodec.VideoHandler`
![image](https://user-images.githubusercontent.com/115556838/214510239-b3cd1651-d214-4ae6-a1cf-c625dbaffa32.png)